sky/serve/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Modules for SkyServe services."""
 import os
 
+from sky.serve import load_balancing_policies as lb_policies
```

Comment:
```suggestion
from sky.serve.load_balancing_policies import LB_POLICIES
```
lets directly expose this ;)